checks/sast_test.go

```diff
@@ -315,10 +339,7 @@ func Test_SAST(t *testing.T) {
 			}
 			res := SAST(&req)
 
-			if res.Score != tt.expected.Score {
-				t.Errorf("Expected score %d, got %d for %v", tt.expected.Score, res.Score, tt.name)
-			}
-			ctrl.Finish()
```

Comment:
just for documenting this, since I forgot I deleted this too

> New in go1.14+, if you are passing a *testing.T into NewController function you no
> // longer need to call ctrl.Finish() in your test methods.

So fine to remove